Plugin.cs

```diff
@@ -153,15 +148,71 @@ private void OnNetworkEvent(IntPtr dataPtr, ushort opCode, uint sourceActorId, u
             if (!this.configuration.alwaysOn && (!this.configuration.smart || !Retainer())) return;
             var listing = MarketBoardCurrentOfferings.Read(dataPtr);
             var i = 0;
-            if (this.configuration.hq && this.items.Single(j => j.RowId == listing.ItemListings[0].CatalogId).CanBeHq)
-            {
+
+            // Load addon ptr from memory
+            var toolTipPtr = pi.Framework.Gui.GetUiObjectByName("ItemDetail", 1) + 0x258;
```

Comment:
Is this `0x258` fixed? It's fine if it changes a few times per expac; I just don't want to have to push a new release every patch/hotfix if it changes all the time.

Reply:
Fixed - yes
That's the address I found in ReClass
I agree - it would be a hassle to maintain it

I tested this a few times a day for the last 2-3 days and its still working
But of course - needed to be furtherly tested

Reply:
Ok, I can release this in testing for now. If the offset still works for the next patch (5.56 I think?), then I will be confident enough to update the main release.